common/components/common/selection/SelectorCollapsible.jsx

```diff
@@ -65,13 +65,9 @@ class SelectorCollapsible<T> extends React.PureComponent<Props<T>, State> {
 
   initializeOptions(props: Props) {
     if(!_.isEmpty(props.options)) {
-
-      let filteredOptions = Object.keys(props.options).filter(function(key) {
-        return props.options[key]['num_times'] > 0;
-      }).map(function(key) {
-        return props.options[key]
-      });
-
+      let filteredOptions = props.options.filter(function(key) {
+          return key.num_times > 0;
```

Comment:
Should we filter out tags with a count of zero?  I guess it reduces clutter/noise in the filter list.  But unless/until more tags are connected to active projects, the smaller subset of initially selected tags could associate the DemocracyLab brand with those arbitrarily selected tags (e.g. could unintentionally pigeonhole D.L. is being only about PHP and MySQL if that is all that the user sees in the available filter list).

Reply:
Yes, that's intentional/intended on my part. This is solving a UX problem with the live site right now where empty filters are causing bad user experience, and based on some conversations I had with some people on the team, this is the best "get it fixed now" solution - and I'd argue it's a solid contender for "get it fixed period" but if it needs to be extended to meet that second goal more thoroughly, that can happen in the future.